src/coreclr/pal/src/exception/signal.cpp

```diff
@@ -746,6 +746,11 @@ static void sigterm_handler(int code, siginfo_t *siginfo, void *context)
 {
     if (PALIsInitialized())
     {
+        char* enable = getenv("COMPlus_EnableDumpOnSigTerm");
```

Comment:
The layer where this was before would've accepted COMPlus or DOTNET. Only a minor back-compat change, not sure if we really mind that given all other env vars in .NET 6 for dumps require the COMPlus prefix.

Reply:
do we need a (separate) issue to eliminate any remaining places where DOTNET_ is not accepted? we presumably don't want to perpetuate the old name in docs, etc as it's meaningless /inconsistent.

Reply:
Ah gotcha. BTW, for completeness, this also breaks anyone using eg COMPLUS_EnableDumpOnSigTerm  (as the change that added centralized support for DOTNET_ also changed the prefix matching so that COMPlus_ was no longer case sensitive on Unix). Not suggesting it matters at all just pointing out.